operator.go

```diff
@@ -660,6 +660,9 @@ func (o *operator) AppendStep(idx int, key string, s map[string]any) error {
 				return fmt.Errorf("invalid dump request: %v", vv)
 			}
 			out := vv["out"]
+			if out == nil {
```

Comment:
@ikawaha 

Thank you!!!!
If you don't mind, could you make a configuration similar to L658-L661?

https://github.com/k1LoW/runn/blob/e5858236dffb8d9836151f9fa067c1fdf166ed01/operator.go#L658-L661

Reply:
Adjusted at 45d4c7a

```
$ cat hello.yml
desc: "hello world!"
steps:
  say_hello:
    desc: "echo hello"
    exec:
      command: echo hello
    test:
      current.exit_code == 0
    dump:
      expr: current.stdout✔ ~/go/src/github.com/k1LoW/runn 
$ go run ./cmd/... run --scopes run:exec hello.yml
Error: failed to append step (hello.yml): invalid dump request: map[expr:current.stdout]
exit status 1
```